code/modules/mob/living/carbon/human/species.dm

```diff
@@ -119,8 +119,10 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/flying_species = FALSE
 	///The actual flying ability given to flying species
 	var/datum/action/innate/flight/fly
-	///The icon used for the wings
+	///Current wings icon
 	var/wings_icon = "Angel"
+	//Dictates which wing icons are allowed for a given species. If count is >1 a radial menu is used to choose between all icons in list
+	var/list/wings_icons = list("Angel")
```

Comment:
Turn this into a string list during initialization. Look for the `string_list` proc for examples in the code.

Reply:
Oooo thank you! will do